Makefile.cache

```diff
@@ -357,6 +357,20 @@ define SAVE_CACHE
 	$(if $(call CHECK_WCACHE_ENABLED,$(1)), $(call SAVE_INTO_CACHE,$(1),$(2)))
 endef
 
+RFS_DEP_FILES := $(wildcard \
+	$(addprefix scripts/, build_debian_base_system.sh prepare_debian_image_buildinfo.sh build_mirror_config.sh) \
+	$(wildcard files/initramfs-tools/*) \
+	$(addprefix files/image_config/, kdump/vmcore-sysctl.conf ntp/ntp-apparmor sonic_release ntp/ntp ntp/ntp-systemd-wrapper \
+		systemd/systemd-udevd/fix-udev-rule-may-fail-if-daemon-reload-command-runs.patch) \
+	$(addprefix files/sshd/, host-ssh-keygen.sh override.conf) \
+	$(addprefix files/dhcp/, rfc3442-classless-routes sethostname sethostname6 graphserviceurl snmpcommunity vrf) \
```

Comment:
Would it be better to use a wildcard/glob expression here, so that if some file gets added/removed, it's automatically added to this list?

Reply:
Good idea, added in https://github.com/sonic-net/sonic-buildimage/pull/15924/commits/2d638f0755d103db0e979ee5b0265f692907e019